CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changes in Fbx Exporter
 
+## [3.2.1-preview.1] - 2020-08-05
+### Added
+- Add an export option to preserve model import settings when overwriting an fbx file.
+- Add the option to export FBX files outside of the Assets folder.
+
+### Changed
+- Changed label name in Autodesk® Maya® from Unity Fbx Namespace to Strip Specific Namespace.
+- Changed Export as FBX Linked Prefab to Export as FBX Prefab Variant.
```

Comment:
For these two, I think "Renamed" instead of Changed

Reply:
Done!